File: config/config-java/src/test/java/quan/config/item/Reward.java
package quan.config.item;

import java.util.*;
import com.alibaba.fastjson.*;
import quan.config.*;

/**
* 奖励<br/>
* Created by 自动生成
*/
public class Reward extends Bean {

    protected final int itemId;

    protected final int itemNum;


    public Reward(JSONObject json) {
        super(json);

        this.itemId = json.getIntValue("itemId");
        this.itemNum = json.getIntValue("itemNum");
    }

    public final int getItemId() {
        return itemId;
    }

    public final int getItemNum() {
        return itemNum;
    }



    @Override
    public String toString() {
        return "Reward{" +
                "itemId=" + itemId +
                ",itemNum=" + itemNum +
                '}';

    }


}

File: quan-cs/Test/Config/Item/Reward.cs
using System;
using System.Collections.Generic;
using System.Collections.Immutable;
using Newtonsoft.Json.Linq;
using Quan.Common;
using Quan.Config;

namespace Test.Config.Item
{
    /// <summary>
	/// 奖励<br/>
	/// 自动生成
	/// </summary>
    public class Reward : Bean
    {

        public readonly int ItemId;

        public readonly int ItemNum;


        public Reward(JObject json) : base(json)
        {
            ItemId = json["itemId"]?.Value<int>() ?? default;
            ItemNum = json["itemNum"]?.Value<int>() ?? default;
        }



        public override string ToString()
        {
            return "Reward{" +
                   "ItemId=" + ItemId.ToString2() +
                   ",ItemNum=" + ItemNum.ToString2() +
                   '}';
        }


    }
}